packages/host/app/lib/current-run.ts

```diff
@@ -122,50 +122,26 @@ export class CurrentRun {
     this.#render = render;
   }
 
-  static async fromScratch(
-    current: CurrentRun,
-    invalidateEntireRealm?: boolean,
-  ): Promise<IndexResults> {
+  static async fromScratch(current: CurrentRun): Promise<IndexResults> {
     let start = Date.now();
     log.debug(`starting from scratch indexing`);
     perfLog.debug(
       `starting from scratch indexing for realm ${current.realmURL.href}`,
     );
-
     current.#batch = await current.#indexWriter.createBatch(current.realmURL);
     let invalidations: URL[] = [];
-    if (invalidateEntireRealm) {
```

Comment:
this was a shortcut that we were doing for creation of new realms that is less efficient than the new copy operation, so we are getting rid of this.